documentation/domains/index.html

```diff
@@ -1222,6 +1222,12 @@
           "description": "The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is FromModel.",
           "type": "string"
         },
+        "maxClusterUnavailable": {
+          "default": 1.0,
+          "description": "The maximum number of cluster members that can be temporarily unavailable. You can override this default on a per cluster basis by setting the cluster\u0027s `maxUnavailable` field. Defaults to 1.",
+          "type": "integer",
+          "minimum": 1.0
```

Comment:
I believe this is generated, but interesting that it shows value of 1.0 even though type is integer.

Reply:
Interesting... I remember updating the CRD generation to get the right form for integers, but I hadn't looked at the swagger.

Reply:
I noticed this, and checked other Integer settings; they are all like this. The format seems only show up in this file. Other generated files (say all json files) are fine.